pkg/flowaggregator/flowaggregator.go

```diff
@@ -724,3 +904,101 @@ func (fa *flowAggregator) createInfoElementForTemplateSet(ieName string, enterpr
 	}
 	return ie, nil
 }
+
+func (fa *flowAggregator) watchConfiguration(stopCh <-chan struct{}) {
+	klog.Info("Watching for FlowAggregator configuration file")
+	for {
+		select {
+		case <-stopCh:
+			return
+		case event, ok := <-fa.configWatcher.Events:
+			if err := fa.handleWatcherEvent(event, ok); err != nil {
+				// if the watcher cannot add mounted configMap or the configMap is not readable, we kill the flow-aggregator pod (serious error)
+				klog.Fatalf("Cannot watch or read configMap: %v", err)
```

Comment:
"configuration file" is more accurate and consistent than "configMap"

Reply:
Updated.